wpilibc/athena/src/IterativeRobot.cpp

```diff
@@ -172,7 +172,6 @@ void IterativeRobot::DisabledPeriodic() {
     printf("Default %s() method... Overload me!\n", __FUNCTION__);
     firstRun = false;
   }
-  delayTicks(1);
```

Comment:
I wondered about that too but the main part of iterativerobot should wait for a new packet anyway...